testsuite/features/step_definitions/command_steps.rb

```diff
@@ -796,9 +796,17 @@
   node.run(cmd, true, DEFAULT_TIMEOUT, 'root', [0, 100, 101, 102, 103, 104, 106])
 end
 
-When(/^I (install|remove) the traditional stack utils (on|from) "([^"]*)"$/) do |action, where, host|
```

Comment:
Please, keep it in a unique step definition.

Reply:
The code for install and remove is different.

It is ridiculous to have it in an unique step just to dijoin cases in that unique step.

Reply:
I'm sorry but I don't agree. 
Please if we need a refresh of the metadata only when installing, express it as I shared bellow.
```
  step %(I refresh the metadata for "#{host}") if action == 'install'
```

Reply:
I will do it, but only to have peace...

I don't understand this compulsion for intricated and complex code.

Reply:
Thank you